Framework/include/QualityControl/CcdbDatabase.h

```diff
@@ -54,9 +54,9 @@ class CcdbDatabase : public DatabaseInterface
 
   void connect(std::string host, std::string database, std::string username, std::string password) override;
   void connect(const std::unordered_map<std::string, std::string>& config) override;
-  void store(std::shared_ptr<o2::quality_control::core::MonitorObject> mo) override;
-  core::MonitorObject* retrieve(std::string taskName, std::string objectName, long timestamp = 0) override;
-  std::string retrieveJson(std::string taskName, std::string objectName) override;
+  void store(std::shared_ptr<o2::quality_control::core::QualityObject> q) override;
```

Comment:
As noted by Piotr, we still need the MonitorObjects to be stored in the database. This is independent from storing the Checks results.

Reply:
I will rename the store to storeQualityObject and storeMonitorObject, as well as retrieve and retrieveJson. Will this be ok or should it be an overloaded store method?